src/js/actions/draganddrop.js

```diff
@@ -64,9 +76,21 @@ define(function (require, exports) {
     deregisterDroppable.writes = [];
 
     /**
+    * Remove many drop targets by a list of keys a drop target by key
+    *
+    * @param {Immutable.Iterable.List<Object>} keys - List of keys to remove
+    * @return {Promise}
+    */
+    var batchDeregisterDroppables = function (keys) {
+        return this.dispatchAsync(events.droppable.DEREGISTER_DROPPABLE, keys);
+    };
+    batchDeregisterDroppables.reads = [];
+    batchDeregisterDroppables.writes = [];
+
+    /**
     * Fire event that dragging started
     *
-    * @param {Immutable.List} dragTarget List of currently dragging items
+    * @param {Immutable.Iterable.List<Object>} dragTarget - List of currently dragging items
```

Comment:
Here, it should just be `@param {Immutable.Iterable.<object>}` or, alternatively, `@param {Immutable.List.<object>}`.